type-definitions/most.d.ts

```diff
@@ -44,7 +44,10 @@ export interface Stream<A> {
   concatMap<B>(f: (a: A) => Stream<B>): Stream<B>;
   mergeConcurrently<B>(concurrency: number): Stream<B>;
   merge(...ss: Array<Stream<A>>): Stream<A>;
+  mergeArray(streams: Array<Stream<A>>): Stream<A>;
   combine<B>(f: (a: A, ...args: Array<any>) => B, ...ss: Array<Stream<any>>): Stream<B>;
+  combineArray<B>(f: (...args: Array<any>) => B, streams: Array<Stream<any>>): Stream<B>;
```

Comment:
Yeah that can work, just PR to my fork and this PR will be updated automatically :+1: 

Thanks!!!